ai21/models/responses/plan_response.py

```diff
@@ -10,6 +10,7 @@ class PlanResponse(AI21BaseModel):
     updated_at: datetime
     assistant_id: str
     code: str
+    schemas: List[dict]
```

Comment:
In other places in the code, you defined schemas as `schemas: List[Dict[str, Any]`, worth aligning to the same convention